data/datasets/__init__.py

```diff
@@ -23,6 +23,7 @@
     "LogicInference_OA": "KK04/LogicInference_OA",
     "oa_dolly_15k": "OllieStanley/oa_dolly_15k",
     "poetry_instruction": "checkai/instruction-poems",
+    "os-stackexchange": "donfu/oa-stackexchange",
```

Comment:
Is `os-stackexchange` here a typo of `oa-stackexchange`? In general we follow the rule of the dict key being the dataset directory name with underscores for separation, so maybe we should change both dict key and directory name to `oa_stackexchange`

Reply:
Makes sense! Implemented it.